sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClient.cs

```diff
@@ -420,8 +423,11 @@ public virtual ServiceBusSender CreateSender(string queueOrTopicName, ServiceBus
 
         /// <summary>
         /// Creates a <see cref="ServiceBusSessionReceiver"/> instance that can be used for receiving
-        /// and settling messages from a specific session-enabled subscription. It uses <see cref="ServiceBusReceiveMode"/> to specify
-        /// how messages are received. Defaults to PeekLock mode. The <see cref="ServiceBusReceiveMode"/> is set in <see cref="ServiceBusReceiverOptions"/>.
+        /// and settling messages from a session-enabled subscription by accepting the next unlocked session that contains Active messages. If there
+        /// are no unlocked sessions with Active messages, then the call will timeout after the configured <see cref="RetryOptions.TryTimeout"/> value and throw
```

Comment:
What will it throw?   I think we're also missing an ending period.   Same for L389

Reply:
Huh? It's on the next line.

Reply:
that was obscured by the validation errors that ran 1/2 my screen.   I see it now.  😄